colibre/registration.py

```diff
@@ -238,6 +244,48 @@
             f"$M_{{\\rm H_2}} / M_{{\\rm HI + H_2}}$ ({aperture_size} kpc)"
         )
 
+        neutral_H_to_baryonic_fraction = neutral_H_mass / (
+            neutral_H_mass + stellar_mass)
+        neutral_H_to_baryonic_fraction.name = (
+            f"$M_{{\\rm HI + H_2}}/((M_*+ M_{{\\rm HI + H_2}})$ ({aperture_size} kpc)"
+        )
+        
+        HI_to_neutral_H_fraction = HI_mass / neutral_H_mass
+        HI_to_neutral_H_fraction.name = (
+            f"$M_{{\\rm HI}}/M_{{\\rm HI + H_2}}$ ({aperture_size} kpc)"
+        )
+
+        H2_to_neutral_H_fraction = H2_mass / neutral_H_mass
+        H2_to_neutral_H_fraction.name = (
+            f"$M_{{\\rm H2}}/M_{{\\rm HI + H_2}}$ ({aperture_size} kpc)"
+        )
+        
+        sf_to_sf_plus_stellar_fraction = sf_mass / (
+            sf_mass + stellar_mass)
+        sf_to_sf_plus_stellar_fraction.name = (
+            f"$M_{{\\rm SF}}/(M_{{\\rm SF}} + M_*)$ ({aperture_size} kpc)"
+        )
+
+        neutral_H_to_sf_fraction = neutral_H_mass / sf_mass
+        neutral_H_to_sf_fraction.name = (
+            f"$M_{{\\rm HI + H_2}}/M_{{\\rm SF}}$ ({aperture_size} kpc)"
+        )
+
+        HI_to_sf_fraction = HI_mass / sf_mass
+        HI_to_sf_fraction.name = (
+            f"$M_{{\\rm HI}}/M_{{\\rm SF}}$ ({aperture_size} kpc)"
+        )
+
+        H2_to_sf_fraction = H2_mass / sf_mass
+        H2_to_sf_fraction.name = (
+            f"$M_{{\\rm H2}}/M_{{\\rm SF}}$ ({aperture_size} kpc)"
+        )
+
+        sf_to_stellar_fraction = H2_mass / sf_mass
```

Comment:
Hi @wmfw23 I have noticed that here the name seems to be inconsistent with what is plotted